paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetConfigurationTest.kt

```diff
@@ -14,9 +14,9 @@ class CustomerSheetConfigurationTest {
     fun `Builder returns a configuration`() {
         val googlePayEnabled = true
         val merchantDisplayName = "Test"
-        val appearance = PaymentSheet.Appearance().copy(
-            typography = PaymentSheet.Typography.default
-        )
+        val appearance = PaymentSheet.Appearance.Builder()
```

Comment:
cc @cttsai-stripe we should also make sure all the docs.stripe.com are updated to use the appearance builder as well!